common/traits/wc_undead_trait.txt

```diff
@@ -0,0 +1,17 @@
+﻿being_undead = {
```

Comment:
I don't see the icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108396118-eff87900-722f-11eb-87b2-5a993f025251.png)

</details>

Reply:
> 
> 
> I don't see the icon.

Fixed in 69bdd7c46661cb7a8f0e151fcaaa0a659fe85086 and tested that it's working even after switching branches.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/108420052-f55bc080-723b-11eb-809a-16d8e0adf97f.png)


</details>